### ISSUE TITLE
Create a presentation for backend

### ISSUE THREAD

**Issue by zcanfes** [bounswe/2021SpringGroup9#347]
### Task: 

Write a template for the things you will say during the presentation. 

### Format (make a table, list etc.):

Text

### Place to put the documentation:

Share with the group and present it on Tuesday. 

### Additional notes:

**Deadline: 16.11.2021**

**Comment by zcanfes** [comment]
I wrote the text: 

We also implemented an API for deleting posts. 
This is a simple but yet an important functionality that gets a story id and deletes it from the database. 

One of the most important functionality is getting all posts which is needed for the homepage. 

Get all posts works by returning all posts and their contents of them from the database. Each post may have multiple tags, locations and images. Tag ids in posts are matched to tag objects in database and their contents are returned. The same process is done for locations and images. 

Let's continue with getting posts by id. 

Get posts works the same as get all posts except it receives a post id to match and return one exact post. Here, it is important to check if a post with that id exists or not. If it doesn't exist, an exception is raised. 

Now, Melih will talk about how we deployed the backend and which tools we used.

**Comment by zcanfes** [comment]
I said the following during the presentation: 

As backend team we prepared the APIs for the functionalities we presented and deployed backend using AWS. For the images, we used AWS S3 buckets. The images are stored there and we keep the urls in the database. We thought this would simplify the work. 

During the implementation, we were in contact with both the Frontend team and the Android team and we plan to continue like this in the future. For the next milestone, login page will be ready and comment, like post functionalities as well as user profile pages will be added. Thank you for listening.
